XRD/xrd_to_pif.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import argparse
+from pypif import pif
+from pypif.obj import *
+
+
+def raw4_to_pif(closed_txt):
+
+    # create chemical system and property array
+    my_pif = ChemicalSystem()
+    my_pif.properties = []
+
+    # Store header index so that iteration can start at next row. Default to False when no header is found.
+    header_line_index = False
+
+    # initialize arrays for 2theta and intensity
+    theta = []
+    intensity = []
+
+    # open text file, read lines, iterate through each line.
+    with open(closed_txt, 'rU') as open_txt:
+        lines = open_txt.readlines()
+        for index, line in enumerate(lines):
```

Comment:
You can close the file now, so drop the following lines down one level of indentation.